src/apps/api/permissions.py

```diff
@@ -25,3 +27,13 @@ def has_object_permission(self, request, view, obj):
 class IsUserAdminOrIsSelf(permissions.BasePermission):
     def has_object_permission(self, request, view, obj):
         return request.user.is_superuser or request.user == obj
+
+
+class IsOrganizationEditor(permissions.IsAuthenticated):
+    def has_object_permission(self, request, view, obj):
+        try:
+            membership = obj.membership_set.get(user=request.user)
+        except Membership.DoesNotExist:
+            return False
+        print(f'Editor Privileges evaluated to {membership.group in membership.EDITORS_GROUP}')
+        return membership.group in membership.EDITORS_GROUP
```

Comment:
Could stick this boolean in a variable of its own and reuse it rather than redefining it

Reply:
That `print` should have gotten removed a while ago. Was it just the evaluation being done twice in a row the problem?